Source/Plugins/Core/com.equella.core/src/com/tle/web/sections/equella/utils/UserLinkSection.java

```diff
@@ -65,6 +66,9 @@
   @PlugKey("userlink.systemuser")
   private static Label LABEL_SYSTEMUSER;
 
+  @PlugKey("userlabel.lastownersuffix")
```

Comment:
Maybe we should stick with the existing namespacing:

```suggestion
  @PlugKey("userlink.lastownersuffix")
```